arda/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Arda Sample</title>
+
+    <link rel="shortcut icon" type="image/png" href="../common/assets/react.png">
+    <link rel="stylesheet" href="../common/css/uikit.almost-flat.min.css">
+    <link rel="stylesheet" href="../common/css/main.css">
+    <script src="build/bundle.js"></script>
```

Comment:
@mizchi please add `build/*.js` of arda to [.gitignore](https://github.com/voronianski/flux-comparison/blob/master/.gitignore#L14).